### ISSUE TITLE
Error: invalid lastal option

### ISSUE THREAD

**Issue by jwidmann** [camillescott/dammit#53]
There seems to be a problem with invalid options for lastal. I have installed last-658 and last-731. Maybe some options have changed. Will dammit take up after this step if i do this by hand?
```
--- Running annotate!

          Transcriptome file: Truncated_Trinity_BoeBlatt.fasta

          Output directory: /home/archibald/Truncated_Trinity_BoeBlatt.fasta.dammit

          [ ] Truncated_Trinity_BoeBlatt.fasta

          [ ] transcriptome_stats:Truncated_Trinity_BoeBlatt.fasta

          [ ] busco:Truncated_Trinity_BoeBlatt.fasta-eukaryota

          [ ] TransDecoder.LongOrfs:Truncated_Trinity_BoeBlatt.fasta

          [ ] hmmscan:longest_orfs.pep.x.Pfam-A.hmm

          [ ] remap_hmmer:longest_orfs.pep.pfam.tbl

          [ ] TransDecoder.Predict:Truncated_Trinity_BoeBlatt.fasta

          [ ] cmscan:Truncated_Trinity_BoeBlatt.fasta.x.Rfam.cm

          [ ] lastal:Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf

          [ ] hmmscan-gff3:Truncated_Trinity_BoeBlatt.fasta.pfam.csv.gff3

          [ ] cmscan-gff3:Truncated_Trinity_BoeBlatt.fasta.rfam.tbl.gff3

Some tasks failed![dammit.annotate:ERROR]
TaskFailed - taskid:lastal:Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf[dammit.annotate:ERROR]
Command failed: '/usr/bin/lastal  -F15 -D148.413159303 /home/archibald/.dammit/databases/aa_seq_euk.fasta.db Truncated_Trinity_BoeBlatt.fasta > Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf' returned 1
[dammit.annotate:ERROR]
Some tasks failed![dammit.annotate:ERROR]
UnmetDependency - taskid:maf_best_hits:Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf-Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf.best.csv[dammit.annotate:ERROR]
lastal:Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf
[dammit.annotate:ERROR]
Some tasks failed![dammit.annotate:ERROR]
UnmetDependency - taskid:maf-gff3:Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf.gff3[dammit.annotate:ERROR]
maf_best_hits:Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf-Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf.best.csv
[dammit.annotate:ERROR]
Some tasks failed![dammit.annotate:ERROR]
UnmetDependency - taskid:gff3-merge:Truncated_Trinity_BoeBlatt.fasta.dammit.gff3[dammit.annotate:ERROR]
maf-gff3:Truncated_Trinity_BoeBlatt.fasta.x.orthodb.maf.gff3
[dammit.annotate:ERROR]
Some tasks failed![dammit.annotate:ERROR]
UnmetDependency - taskid:fasta-annotate:Truncated_Trinity_BoeBlatt.fasta.dammit.fasta[dammit.annotate:ERROR]
gff3-merge:Truncated_Trinity_BoeBlatt.fasta.dammit.gff3
[dammit.annotate:ERROR]

```
Best wishes

**Comment by camillescott** [comment]
Looking into this further. I think this shows up if LAST is compiled without openmp; I'll be switching over to the gnu-parallel version described in the docs soon (http://last.cbrc.jp/doc/last-parallel.html). In the meantime, yes, you *should* be able to continue by running that task manually, or, by running with n_threads = 1, canceling, and running again with however many threads you please.

**Comment by jwidmann** [comment]
Thanks for your reply. When I try running with n_threads=1 it just gives me directly the same output as before - do I have to delete a log file or something like that? Running lastal manually also doesn't work (also directly the error output). Any suggestions?

**Comment by Victaphanta** [comment]
DAMMIT - Same problem here.  Installed all dependencies, followed by dammit itself - all appropriately in PATH, and then ran >dammit dependencies and all is in order except LAST:
lastal: invalid option -- '-'
lastal: bad option
......
          [ ] LAST... [1]
          [x] crb-blast

--- Dependency results

          LAST missing

Also tried LAST-737 which compiled with -std=c++11, which apparently is needed for multithreading.  Still no cookie.

Any help would be appreciated.
Look forward to testing you pipeline.  Thanks for making it available

**Comment by camillescott** [comment]
This has now been fixed in v0.2.9. @Victaphanta , I believe your issue is something else, so I'll make a new issue for it.
